components/Snackbar/src/MDCSnackbarManager.m

```diff
@@ -498,19 +491,49 @@ - (void)removeSuspensionIdentifierMainThread:(NSUUID *)identifier forCategory:(N
 
 #pragma mark - Public API
 
-@implementation MDCSnackbarManager
+@interface MDCSnackbarManager ()
+@property(nonnull, nonatomic, strong) MDCSnackbarManagerInternal *internalManager;
+@end
 
-+ (void)setDelegate:(id<MDCSnackbarManagerDelegate>)delegate {
-  MDCSnackbarManagerInternal *manager = [MDCSnackbarManagerInternal sharedInstance];
-  manager.delegate = delegate;
+@implementation MDCSnackbarManager {
+  UIColor *_snackbarMessageViewBackgroundColor;
+  UIColor *_snackbarMessageViewShadowColor;
+  UIColor *_messageTextColor;
+  UIFont *_messageFont;
+  UIFont *_buttonFont;
+  NSMutableDictionary<NSNumber *, UIColor *> *_buttonTitleColors;
+  BOOL _mdc_adjustsFontForContentSizeCategory;
+  BOOL _shouldApplyStyleChangesToVisibleSnackbars;
 }
 
-+ (id<MDCSnackbarManagerDelegate>)delegate {
-  MDCSnackbarManagerInternal *manager = [MDCSnackbarManagerInternal sharedInstance];
-  return manager.delegate;
+
++ (instancetype)defaultManager {
+  static MDCSnackbarManager *kDefaultManager;
+  static dispatch_once_t onceToken;
+  dispatch_once(&onceToken, ^{
+    kDefaultManager = [[MDCSnackbarManager alloc] init];
+  });
+  return kDefaultManager;
```

Comment:
[style] We typically reserve `k` for file-level constants. Consider dropping the `k` here.

Reply:
Sure, how do we mark static variables that are scoped to a method/function?

Reply:
Also, I found that 3/4 of our uses of `dispatch_once` use the `k` prefix for the static variable being initialized. I was previously told not to use `s_` for statics (we prefer `k` for `static const`s).

* [MDCTypographySchemeFontListExampleViewController.m](https://github.com/material-components/material-components-ios/blob/develop/components/schemes/Typography/examples/MDCTypographySchemeFontListExampleViewController.m) (authored by me)
* [AnimationTimingExampleSupplemental.m](https://github.com/material-components/material-components-ios/blob/develop/components/AnimationTiming/examples/supplemental/AnimationTimingExampleSupplemental.m) (authored by you)

Reply:
We typically just name it like a local variable, so `defaultManager` in this case.

Reply:
:( Sounds confusing. Alright.